src/symbolic/symbol_scope.mli

```diff
@@ -0,0 +1,35 @@
+type token
+
+type t
+
+val symbol : Smtml.Symbol.t -> token
```

Comment:
I think we should make this have type:

```ocaml
val symbol: Smtml.Symbol.t -> t -> t
```

So that we can completely hide the `token` type.